Formula/mpd.rb

```diff
@@ -1,6 +1,7 @@
 class Mpd < Formula
   desc "Music Player Daemon"
   homepage "http://www.musicpd.org/"
+  revision 1
```

Comment:
I was a bit unsure if I needed a new revision (this is my first PR for Homebrew). I used [Formulae Revisions from the Formula Cookbook](https://github.com/Homebrew/brew/blob/master/share/doc/homebrew/Formula-Cookbook.md#formulae-revisions) as reference ("(...) formulae updates that don’t include a version bump"). 

My motivation was that people who had already installed the 0.19.14 version of mpd should get the updated formula before a new mpd version is released upstream. But I may have misunderstood how these things should be handled.